lib/widgets/components/shop_section.dart

```diff
@@ -72,7 +72,7 @@ class ShopSection extends StatelessWidget {
                   Navigator.push(
                     context,
                     MaterialPageRoute(
-                      builder: (context) => const RedeemVoucherPage(),
+                      builder: (context) => RedeemVoucherPage(),
```

Comment:
Should be `const` after you do the changes requested in `redeeme_voucher_page.dart`